test/new-e2e/go.mod

```diff
@@ -7,6 +7,7 @@ go 1.21
 // TODO: Implement hard check in CI
 
 replace (
+	github.com/DataDog/test-infra-definitions => /home/kjn/CodeBase/test-infra-definitions
```

Comment:
Required just for the moment. depends on https://github.com/DataDog/test-infra-definitions/pull/582